src/modules/SuSEFirewallCMDLine.rb

```diff
@@ -1599,6 +1634,40 @@ def Run
       nil
     end
 
+  private
+    # Returns true if FirewallD is the running backend
+    def firewalld?
+      return true if SuSEFirewall.is_a?(Yast::SuSEFirewalldClass)
```

Comment:
@jreidinger I don't understand that. Could you explain what you mean? SuSEFirewall is an object which is instantiated in SuSEFirewall.rb. It's type can either be SuSEFirewalldClass or SuSEFirewall2Class. Both of these classes are being pulled in in SuSEFirewall.rb. So I am not sure what's wrong with that...